src/Features/Navigation/Screen.php

```diff
@@ -99,6 +124,24 @@ public static function is_woocommerce_page() {
 		return false;
 	}
 
+	/**
+	 * Check if a given taxonomy is a WooCommerce related taxonomy.
+	 *
+	 * @param string $taxonomy Taxonomy.
+	 * @return bool
+	 */
+	public static function is_woocommerce_taxonomy( $taxonomy ) {
+		if ( in_array( $taxonomy, array( 'product_cat', 'product_tag' ), true ) ) {
```

Comment:
If an extension added a taxonomy, would it need to be included in this array via filter?

Reply:
No, they would still register via `register_taxonomy()`.